ci-operator/step-registry/rhsi/run-tests/rhsi-run-tests-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -u
+set -e
+set -o pipefail
+
+export PUBKUBECONFIG=$KUBECONFIG
+
+export PRIVKUBECONFIG=$KUBECONFIG
+
+cd /app
+
+status=0
+
+./run-test.sh || status="$?" || :
+
+mkdir -p $ARTIFACT_DIR/result
+
+# Copy results to ARTIFACT_DIR
+cp -r /result/* $ARTIFACT_DIR/result 2>/dev/null || :
+cp -r /tmp/test.out $ARTIFACT_DIR/result 2>/dev/null || :
+
+# Prepend junit_ to result xml files
+rename '/junit' '/junit_rhsi' ${ARTIFACT_DIR}/result/*.xml 2>/dev/null || :
```

Comment:
@smatula I see xmls are not getting renamed [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36424/rehearse-36424-periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws/1706360517720805376/artifacts/rhsi-interop-aws/rhsi-run-tests/artifacts/result/) can you recheck

Reply:
@chetna14manku Updated code. rename did not exist on container. Filenames  are now getting updated.